src/notes_common.h

```diff
@@ -5,7 +5,8 @@
 #include "formats.h"
 
 #define SALTLEN                 16
-#define BLOBLEN                 24
+#define BLOBLEN                 40  // 24 is the actual size, making it 40 for compatibility with "fvde" kernel
+#define BLOBLEN_ACTUAL          24
```

Comment:
A more handsome solution would be to pass `-DBLOBLEN=%d` to build options and change things in the kernel accordingly (eg. `int32_t n = 4;  // len(C) - 1` -> `int32_t n = BLOBLEN / 8 - 1;`).

But this works too...

Reply:
I have implemented this now.

Reply:
To be future-safe, the `uint64_t R[5];` should be `uint64_t R[BLOBLEN / 8];`.

Reply:
Yep - I missed that.